agent/__main__.py

```diff
@@ -21,14 +22,37 @@ def main():
     }
 
     patches = {
-        'openssh-empty-password': 'OpenSSH: Disable logins with empty password',
-        'openssh-root-login': 'OpenSSH: Disable root login',
-        'openssh-password-auth': 'OpenSSH: Disable password authentication',
-        'openssh-agent-forwarding': 'OpenSSH: Disable agent forwarding',
-        'openssh-protocol': '\tOpenSSH: Force protocol version 2'
+        'openssh-empty-password':
+            ('OpenSSH: Disable logins with empty password', 'PermitEmptyPasswords'),
+        'openssh-root-login':
+            ('OpenSSH: Disable root login', 'PermitRootLogin'),
+        'openssh-password-auth':
+            ('OpenSSH: Disable password authentication', 'PasswordAuthentication'),
+        'openssh-agent-forwarding':
+            ('OpenSSH: Disable agent forwarding', 'AllowAgentForwarding'),
+        'openssh-protocol':
+            ('\tOpenSSH: Force protocol version 2', 'Protocol'),
```

Comment:
What's the purpose of this '\t' here?

Reply:
To align the output in the console. Otherwise it's hard to read.